projects/core/src/cms/store/selectors/page.selectors.spec.ts

```diff
@@ -9,16 +9,29 @@ import * as fromSelectors from '../selectors/page.selectors';
 import { EntityLoaderState, LoaderState } from '../../../state';
 import { ContentSlotData } from '../../model/content-slot-data.model';
 import { Page } from '../../model/page.model';
-import { CmsComponent, PageType } from '../../../occ/occ-models/index';
+import { PageType } from '../../../occ/occ-models/index';
 import { PageContext } from '../../../routing/models/page-context.model';
+import { ContentSlotComponentData } from '@spartacus/core';
 
 describe('Cms PageData Selectors', () => {
   let store: Store<StateWithCms>;
 
-  const components: CmsComponent[] = [
-    { uid: 'comp1', typeCode: 'SimpleBannerComponent' },
-    { uid: 'comp2', typeCode: 'CMSLinkComponent' },
-    { uid: 'comp3', typeCode: 'NavigationComponent' }
+  const components: ContentSlotComponentData[] = [
+    {
+      uid: 'comp1',
+      typeCode: 'SimpleBannerComponent',
```

Comment:
Are `typeCode`s needed in mock data? If yes, shouldn't we add one example when typeCode is different than flexType?

Reply:
It's not needed.